app/models/data_resources/generic_item.rb

```diff
@@ -0,0 +1,67 @@
+class GenericItem < ApplicationRecord
+  include FilterByRole
+  has_ancestry orphan_strategy: :destroy
+
+  validates_presence_of :generic_type
+  store :payload, coder: JSON
+
+  belongs_to :data_provider
+  has_one :external_reference, as: :external, dependent: :destroy
+  has_many :accessibility_informations, as: :accessable, dependent: :destroy
```

Comment:
wieso sind die hier `has_many` statt einzeln? die anderen datentypen haben immer nur `has_one :accessibility_information, as: :accessable, dependent: :destroy`

Reply:
weil ich bei allem in genericItem immer ein has_many draus mache, wozu die künstliche Beschneidung das es nur eins geben kann?